BigBaseV2/src/views/vehicle/view_lsc.cpp

```diff
@@ -742,9 +748,7 @@ namespace big
 							selected_color = color;
 							headlight_color = color;
 
-							g_fiber_pool->queue_job([] {
-								VEHICLE::SET_VEHICLE_XENON_LIGHTS_COLOR_(player_vehicle, headlight_color);
-							});
+							VEHICLE::SET_VEHICLE_XENON_LIGHTS_COLOR_(player_vehicle, headlight_color);
```

Comment:
Calling natives outside of script threads is detected.

Use the fiber pool please.